regulations/static/regulations/js/source/views/main/child-view.js

```diff
@@ -104,20 +105,23 @@ var ChildView = Backbone.View.extend({
                 if (_.isEmpty(this.activeSection) || (this.activeSection !== this.$sections[i].id)) {
                     this.activeSection = this.$sections[i][0].id;
                     this.$activeSection = $(this.$sections[i][0]);
-                    // **Event** trigger active section change
-                    HeaderEvents.trigger('section:open', this.activeSection);
-                    MainEvents.trigger('paragraph:active', this.activeSection);
-
-                    if (typeof window.history !== 'undefined' && typeof window.history.replaceState !== 'undefined') {
-                        // update hash in url
-                        window.history.replaceState(
-                            null,
-                            null,
-                            window.location.origin + window.location.pathname + window.location.search + '#' + this.activeSection
-                        );
+                    if (this.$activeSection.is(':visible')) {
```

Comment:
It might help to comment out the visibility check locally and see what happens. We don't run into errors, but we do update the current toc item when we shouldn't. The dom isn't actually all that different here, since all the read mode content is still here--it's just hidden in write mode.